vcsclient/gitlab.go

```diff
@@ -374,6 +374,10 @@ func (client *GitLabClient) UnlabelPullRequest(ctx context.Context, owner, repos
 	return err
 }
 
+func (client *GitLabClient) UploadCodeScanning(ctx context.Context, owner string, repository string, branch string, scan string) (string, error) {
+	return "", errCodeScanningNotSupported
```

Comment:
It looks like the error returned here is: "code scanning is not supported on Bitbucket"

Reply:
done